<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sp_rects</class>
 <widget class="QMainWindow" name="sp_rects">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1010</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>sp_rects</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#sp_rects {
background-color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1010</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1010</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>5</y>
      <width>210</width>
      <height>790</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="info_button">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>750</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#info_button {
	background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}

#info_button:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}</string>
     </property>
     <property name="text">
      <string>Usage Info</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="source_rec">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>540</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="x1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>543</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>source:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="y2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
    <widget class="QTextBrowser" name="show_graph">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="dest_rec">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>540</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>44</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>y1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>The List of Rectangles</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>14</y>
       <width>23</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>x2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>48</x>
       <y>14</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>x1:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="show_coordinates">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>191</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="x2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>44</y>
       <width>24</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>y2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>543</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>dest:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_graph_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>680</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#show_graph_button {
	background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}

#show_graph_button:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}</string>
     </property>
     <property name="text">
      <string>Show Shortest Path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_rect_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#add_rect_button {
	background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}

#add_rect_button:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}</string>
     </property>
     <property name="text">
      <string>Add Rectangle</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="y1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>index:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="deleting_rect">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>207</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_rect_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#delete_rect_button {
	background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}

#delete_rect_button:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}</string>
     </property>
     <property name="text">
      <string>Delete Rectangle</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upload_file_button">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#upload_file_button {
	background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}

#add_rect_button:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}</string>
     </property>
     <property name="text">
      <string>Upload File</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
